Documentation/Wiki/Release-Notes.md

```diff
@@ -11,9 +11,26 @@ This page is a curated list of the release notes for releases after 0.20170619.4
 See the [the BuildXL Release Management page for the Production environment](https://dev.azure.com/mseng/domino/_release?definitionId=21&definitionEnvironmentId=112&_a=environment-summary) for full commit-level details for what is included in each build.
 When a BuildXL developer implements a feature, fixes an important bug, solves an issue brought up by a customer, or makes any other notable change, they are encouraged to add an entry here.
 
-# Upcoming release: 
+# 0.1.0-20191011.9.1 (Release [43540](https://dev.azure.com/mseng/Domino/_releaseProgress?_a=release-pipeline-progress&releaseId=43540)). Released 10/16/2019
+- Support for partial preserve outputs
+- Symbol daemon runner is configured for CloudBuild environment
+- New QTest version (19.10.5.1051)
+- Fix IPC pip ordering for BinaryGraph
+- CBdependencies feature an opt-out instead of opt-in
+- Improve time to first pip for distributed builds
+- Enforce weak fingerprint augmentation for MSBuild-scheduled pips
+- Use xattrs to mark shared opaque outputs on Mac
+
+# 0.1.0-20191003.4 (Release [42310](https://dev.azure.com/mseng/Domino/_releaseProgress?_a=release-pipeline-progress&releaseId=42310)). Released 10/09/2019
 - Removed old Microsoft.ContentStoreApp.exe from deployment (replaced with ContentStoreApp.exe)
-- ...
+- Proper SIGILL handling on macOS

```

Comment:
Sorry I forgot to do it.

Reply:
it is not clear from our dri doc :) so, i updated the doc as well

Reply:
It was clear; I saw it the instructions, and I saw Oleksii do it the week before, so I was going to do it, but I just forgot.